spot_driver_cpp/launch/spot_image_publisher.launch.xml

```diff
@@ -0,0 +1,28 @@
+<launch>
+  <!-- Args for required parameters: -->
+  <arg name="address" />
+  <arg name="username" />
+  <arg name="password" />
```

Comment:
Are these required?  What happens if they're specified in environment parameters?

Reply:
I added env var parsing to set the default values for these. This launch file is an alternative entrypoint for how this new node will ultimately be run in `spot_driver.launch.py` (see the follow-on work in #162 for how that will look), so these launch arguments won't be used in that case.

Reply:
I ended up deleting this launch file.